doc/release-notes.md

```diff
@@ -50,33 +53,246 @@ is not recommended.
+----------------------------------
+The masternode hard fork signal special transaction has been added as the first
+step in enabling an improved hard fork mechanism. This enhancement enables
+future hard forks to be activated quickly and safely without any
+“race conditions” if miners and masternodes update at significantly different
+speeds. Effectively there will be a masternode signal on chain in addition to
+the miner one to ensure smooth transitions. Details of the enhanced hard fork
+system are provided in [DIP-0023](https://github.com/dashpay/dips/blob/master/dip-0023.md).
+
+Network improvements
+--------------------
+We implemented and backported implementations of several improvement proposals.
+You can read more about implemented changes in the following documents:
+- [`DIP-0025`](https://github.com/dashpay/dips/blob/master/dip-0025.md): Compressed headers.
+- [`BIP 155`](https://github.com/bitcoin/bips/blob/master/bip-0155.mediawiki): The 'addrv2' and 'sendaddrv2' messages which enable relay of Tor V3 addresses (and other networks).
+- [`BIP 158`](https://github.com/bitcoin/bips/blob/master/bip-0158.mediawiki): Compact Block Filters for Light Clients.
+
+KeePass support removed
+-----------------------
+Please make sure to move your coins to a wallet with a regular passphrase.
+
+Wallet changes
+--------------
+We continued backporting wallet functionality updates. Most notable changes
+are:
+- Added support for empty, encrypted-on-creation and watch-only wallets.
+- Wallets can now be created, opened and closed via a GUI menu.
+- No more `salvagewallet` option in cmd-line and Repair tab in GUI. Check the
+`salvage` command in the `dash-wallet` tool.
+
+Indexes
+-------
+The transaction index is moved into `indexes/` folder. The migration of the old
+data is done on the first run and does not require reindexing. Note that the data
+in the old path is removed which means that this change is not backwards
+compatible and you'll have to reindex the whole blockchain if you decide to
+downgrade to a pre-v18.0.0 version.
+
+Remote Procedure Call (RPC) Changes
+-----------------------------------
+Most changes here were introduced through Bitcoin backports mostly related to
+the deprecation of wallet accounts in DashCore v0.17 and introduction of PSBT
+format.
+
+The new RPCs are:
+- `combinepsbt`
+- `converttopsbt`
+- `createpsbt`
+- `decodepsbt`
+- `deriveaddresses`
+- `finalizepsbt`
+- `getblockfilter`
+- `getdescriptorinfo`
+- `getnodeaddresses`
+- `getrpcinfo`
+- `joinpsbts`
+- `listwalletdir`
+- `quorum rotationinfo`
+- `scantxoutset`
+- `submitheader`
+- `testmempoolaccept`
+- `utxoupdatepsbt`
+- `walletcreatefundedpsbt`
+- `walletprocesspsbt`
+
+The removed RPCs are:
+- `estimatefee`
+- `getinfo`
+- `getreceivedbyaccount`
+- `keepass`
+- `listaccounts`
+- `listreceivedbyaccount`
+- `move`
+- `resendwallettransactions`
+- `sendfrom`
+- `signrawtransaction`
+
+Changes in existing RPCs introduced through bitcon backports:
```

Comment:
```suggestion
Changes in existing RPCs introduced through bitcoin backports:
```

Reply:
sigh..

Reply:
I didn't catch that one - my spell checker did :smile: